tests/conftest.py

```diff
@@ -25,13 +25,13 @@
 """
 import sys
 import os
-from pkg_resources import resource_filename
+from importlib.resources import files
```

Comment:
For `climate-explorer-backend` I did a [drop in replacement](https://github.com/pacificclimate/climate-explorer-backend/pull/249/commits/456c42fdf3da4f9c98c4381c943ad20ba564fbd0) for `resource_filename()` to encapsulate all of the funky syntax. Maybe you could do something like that and add it `modelmeta.__init__.py`?